docs/AppFramework.md

```diff
@@ -1,30 +1,36 @@
 # App Framework Resource Guide
 
-The Splunk Operator provides support for Splunk App and Add-on deployment using the App Framework (Beta Version). The App Framework specification adds support for S3-compatible app repository, authentication, and supports specific Splunk Enterprise cluster and standalone [custom resources](https://splunk.github.io/splunk-operator/CustomResources.html) (CR).
+The Splunk Operator provides support for Splunk app and add-on deployment using the App Framework. The App Framework specification supports configuration management using the Splunk Enterprise cluster and standalone [custom resources](https://splunk.github.io/splunk-operator/CustomResources.html) (CR). 
 
 ### Prerequisites
 
 Utilizing the App Framework requires:
 
-* An Amazon S3 or S3-API-compliant remote object storage location. App framework requires read-only access to the path containing the apps.
-* The remote object storage credentials via a secret, or an IAM role.
-* Splunk Apps and Add-ons in a .tgz or .spl archive format.
-* Connections to the remote object storage endpoint need to be secure using a minimum version of TLS 1.2.
+* An Amazon S3 or S3-API-compliant remote object storage location. The App framework requires read-only access to the path used to host the apps.
+* The remote object storage credentials provided as a secret, or in an IAM role.
+* Splunk apps and add-ons in a .tgz or .spl archive format.
+* Connections to the remote object storage endpoint need to be secured using a minimum version of TLS 1.2.
+* A persistent storage volume and path for the Operator Pod. 
 
+#### What is required to upgrade to the latest App Framework?
+1. Upgrade the Operator to the latest release. The Operator pod will restart. The apps and add-ons already manged by the App Framework will remain deployed, and their status won't change. Any new apps, or changes to existing apps will be ignored.
+2. Update the Operator to add a persistent storage volume. The Operator pod will restart. 
+  * Example: `insert an example yaml to porvide a storage path to the operator pod`
```

Comment:
@sgontla Do you have a handy example to use? Thanks!

Reply:
I don't think that's been normalized. See: 667741767953.dkr.ecr.us-west-2.amazonaws.com/sgontla/
Can I have a cleaned-up example? Thanks!